app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UserMailer < ApplicationMailer
+  def activation_needed_email(user)
```

Comment:
What do you think about the name: `activation`, since it is already known to be an email (Otherwise it would not be in the UserMailer).